src/components/glossary/GlossaryPage.js

```diff
@@ -1,63 +1,65 @@
-import React, {Component} from "react";
-import "./GlossaryPage.css"
-import { Search } from "../common/index";
-import { Link } from "react-router-dom";
+import React from 'react';
+import './GlossaryPage.css';
+import { Search } from '../common/index';
+import { Link } from 'react-router-dom';
 
 import glossarioLogo from '../../../assets/images/glossario-logo.svg';
 
 import terms from '../../lib/data';
-import SearchResults from "./results/SearchResults";
+import SearchResults from './results/SearchResults';
 
 const DayPhrase = ({ entry }) => {
-    return (
-        <span className={"glossary__day-phrase"}>
-            Você sabe o que é <Link className="emphasis pointer-hover light-accent lighter-hover" to={`/${entry}`}>{ entry }</Link>?
-        </span>)
+  return (
+    <span className={'glossary__day-phrase'}>
+      Você sabe o que é{' '}
```

Comment:
Esse `{' '}` tem alguma finalidade?

Reply:
Hi @fanny, I appreciate the suggestions! They're super helpful in learning a bit more about ideal ways to structure code in React :)

Reply:
You're welcome!